src/thor/triplegbuilder.cc

```diff
@@ -1104,13 +1104,9 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
     }
   }
 
-  // Set mean elevation if requested (set to kNoElevationData if unavailable)
+  // Set mean elevation if requested (will be kNoElevationData if unavailable)
   if (controller(kEdgeMeanElevation)) {
-    if (graphtile->header()->has_elevation()) {
-      trip_edge->set_mean_elevation(edgeinfo.mean_elevation());
-    } else {
-      trip_edge->set_mean_elevation(kNoElevationData);
-    }
+    trip_edge->set_mean_elevation(edgeinfo.mean_elevation());
```

Comment:
this was redundant for elevation. the `set_mean_elevation` will handle the case when there's no elevation set

Reply:
its redundant now that you fixed it! it wasnt redundant in the past, previously this protected us from getting `1234` in the response